components/agents/org.wso2.carbon.identity.entitlement.proxy/src/main/org/wso2/carbon/identity/entitlement/proxy/wsxacml/WSXACMLEntitlementServiceClient.java

```diff
@@ -76,6 +78,7 @@
 import javax.xml.parsers.DocumentBuilder;
 import javax.xml.parsers.DocumentBuilderFactory;
 import java.io.*;
```

Comment:
imports with * should be fixed with individual imports. IDE Optimize imports option can be used.